examples/deploy/terraform/cluster/main.tf

```diff
@@ -44,7 +45,6 @@ locals {
 # the following annotation to the `external-dns` service account:
 # `eks.amazonaws.com/role-arn: <<module.irsa_external_dns.irsa_role>>`
 module "irsa_external_dns" {
-  count               = var.irsa_external_dns != null && var.irsa_external_dns.enabled ? 1 : 0
```

Comment:
why did this change?

Reply:
```
"The configuration of module.irsa_external_dns has its own local configuration for aws, and so it cannot accept an overridden configuration provided by the root module.",
```

Want to enforce the provider config `use_fips_endpoint` at the module level. Given the IRSA module does not/should not create anything without additional settings(i.e `var.irsa_external_dns.enabled` ). Not sure if it will be possible once integrated with how p-a consumes the module